vnext/Directory.Build.props

```diff
@@ -13,6 +13,9 @@
 
   </PropertyGroup>
 
+  <!-- Initialize $(ReactNativeWindowsDir) before it is used below. -->
+  <Import Project="PropertySheets\ReactPackageDirectories.props" />
```

Comment:
Why here, and not in `PropertySeets\React.Cpp.props`?

Reply:
Noted. It initializes directory properties.

Reply:
We need the $(ReactNativeWindowsDir) property in this file and changing React.Cpp.props will not make any difference because React.Cpp.props is not imported by the Directory.Build.props, It causes the build and target folders to be relative and fail compilation of new user projects.